go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/operator-framework/operator-marketplace
 
-go 1.19
+go 1.21
```

Comment:
we were requiring go 1.19 but building on the 1.20 base image?

Reply:
Apparently